package-lock.json

```diff
@@ -1311,7 +1311,7 @@
     },
     "ansi-escapes": {
       "version": "3.1.0",
-      "resolved": "https://registry.npmjs.org/ansi-escapes/-/ansi-escapes-3.1.0.tgz",
+      "resolved": "http://registry.npmjs.org/ansi-escapes/-/ansi-escapes-3.1.0.tgz",
```

Comment:
How those changes happened to be here? I don't see any package.json updates.

Reply:
@awalkowiak  I don't really know for this one, I got that pulling develop and doing npm install, I will have a look at the history of develop